ipu-plugin/buildkit.toml

```diff
@@ -0,0 +1,2 @@
+[registry."<HostIP>:5000"]
+  http = true
```

Comment:
Are we adding any documentation or some comments in Makefile on how to use this file?

Reply:
I will add a comment to the Makefile on how to use this file